packages/ui/components/AudioView/AudioView.tsx

```diff
@@ -0,0 +1,64 @@
+import { FunctionComponent, useState, useEffect, useRef } from 'react'
+import { IReportFile } from '../../domain/ReportFile'
+import { MediaControls } from '../MediaControls/MediaControls';
+import { AudioVisualization } from './AudioVisualization'
+
+type Props = {
+  file: IReportFile
+};
+
+export const AudioView: FunctionComponent<Props> = ({file}) => {
+  const [canPlay, handleCanPlay] = useState<boolean>(false)
+  const [currentPercentage, handlePercentage] = useState<number>(0)
+  const fileRef = useRef<HTMLVideoElement | null>(null)
+
+  useEffect(() => {
+    if (!fileRef) {
+      return
+    }
+
+    fileRef.current.addEventListener('canplay', () => {
+      handleCanPlay(true)
+    })
+
+    fileRef.current.addEventListener("timeupdate", () => {
+      handlePercentage((fileRef.current.currentTime / fileRef.current.duration) * 100)
+    })
+  }, [])
```

Comment:
If an event is added it should be removed in the function that returns useEffect. Something like:
```suggestion
  return () => fileRef.current.removeEventListener("timeupdate", handleTimeUpdate);
  }, [])
```